build.gradle

```diff
@@ -306,6 +306,7 @@ dependencies {
   testCompile 'pl.pragmatists:JUnitParams:1.1.1'
   testCompile group: 'org.mockito', name: 'mockito-core', version: '3.12.4'
   testCompile group: 'io.github.artsok', name: 'rerunner-jupiter', version: '2.1.6'
+  testCompile group: 'com.github.hmcts', name: 'ccd-case-document-am-client', version: '1.7.1'
```

Comment:
strange that this is needed?
Is it needed?

Reply:
The test wouldn't compile without it. It couldn't find the UploadResponse object. If you can find a way round it without this then let me know.